ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -401,14 +401,13 @@ module Configuration = struct
       else
         hostname
     in
-    Printf.sprintf "InitiatorName=%s\nInitiatorAlias=%s\n" iqn hostname_chopped
+    [iqn; hostname_chopped]
 
   let set_initiator_name iqn =
     let hostname = Unix.gethostname () in
     let config_file = make_initiatorname_config iqn hostname in
```

Comment:
The names `config_file` and `make_initiatorname_config` don't make sense anymore, which is confusing. I suggest `let args = make_set_initiator_args ...`.